bands/templates/bands/bio.html

```diff
@@ -9,41 +9,42 @@
 <br/> <br/>
 
 <form method="post" action="bio" class="form-horizontal" role="form" enctype="multipart/form-data">
-    <input type="hidden" name="id" value="{{ band.id }}" />
+    <input type="hidden" name="band_id" value="{{ band.id }}" />
+    <input type="hidden" name="id" value="{{ band.bio.id|default:None }}" />
     {% csrf_token %}
 
     <div class="form-group">
         <label class="col-sm-2 control-label">Short Bio:</label>
         <div class="col-sm-6">
-            <textarea class="form-control" name="bio">{{ band.bio }}</textarea>
+            <textarea class="form-control" name="desc">{{ band.bio.desc }}</textarea>
```

Comment:
The very first time that a band fills out this checklist, ``band.bio`` won't exist, which means that this field will be blank by default.  Which is fine when the form is first presented, but this means that if they fill out the form and submit it and have an error, the fields they already filled out won't be there.  You should say ``band_bio.desc`` instead of ``band.bio.desc`` to prevent this.